components/org.wso2.carbon.identity.conditional.auth.functions.analytics/pom.xml

```diff
@@ -109,6 +109,12 @@
             <groupId>org.wso2.orbit.org.apache.httpcomponents</groupId>
             <artifactId>httpasyncclient</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.openjdk.nashorn</groupId>
+            <artifactId>nashorn-core</artifactId>
+            <version>15.3</version>
```

Comment:
Shall we move the dependency version to main pom file? Check other places too.

Reply:
addressed with [https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/103/commits/0317f138dcae64218e10e5efc44ec3eed8ad2a1c](https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/103/commits/0317f138dcae64218e10e5efc44ec3eed8ad2a1c)